packages/openapi-generator/src/options/options-per-service.spec.ts

```diff
@@ -75,7 +80,139 @@ describe('getServiceOptions', () => {
       getServiceOptions(optionsPerService, 'some/input.json', 'serviceName')
     ).toEqual({
       packageName: 'customPackageName',
-      directoryName: 'serviceName'
+      directoryName: 'serviceName',
+      serviceName: 'serviceName'
+    });
+  });
+});
+
+describe('getOptionsPerService', () => {
+  it('builds PerService config without options per service.', async () => {
+    await expect(
+      getOptionsPerService(
+        [{ absolutePath: '/user/path/service', relativePath: 'path/service' }],
+        {} as ParsedGeneratorOptions
+      )
+    ).resolves.toMatchInlineSnapshot(`
+            Object {
+              "path/service": Object {
+                "directoryName": "service",
+                "packageName": "service",
+                "serviceName": "service",
+              },
+            }
+          `);
+  });
+
+  it('builds PerService config with options per service.', async () => {
```

Comment:
Probably a left over from the previous renaming, but from the name I really don't know what this test does. 
```suggestion
  it('builds options per service with existing config file', async () => {
```

Consider removing the notion of `PerService config` from the other tests as well, it implies that we have a class that is named like this.

Reply:
After looking at this for a while I think the reason is, that for me "options per service" is what I want to get out of this function.